services/net/content/ContentManager.cs

```diff
@@ -260,8 +260,7 @@ private async Task<ConsumerAction> HandleMessageAsync(ConsumeResult<string, Sour
                 {
                     // TODO: Handle different storage locations.
                     // If the source content references a connection then fetch it to get the storage location of the file.
-                    var connection = model.ConnectionId.HasValue ? await _api.GetConnectionAsync(model.ConnectionId.Value) : null;
-                    var fullPath = Path.Combine(_options.VolumePath, connection?.GetConfigurationValue("path")?.MakeRelativePath() ?? "", model.FilePath.MakeRelativePath());
+                    var fullPath = Path.Combine(_options.VolumePath, model.FilePath.MakeRelativePath());
```

Comment:
Each content record with a file has a path that does not include the mapped volume.